### PR TITLE
Add configure step

### PR DESCRIPTION
**Pull request by mcbattirola**
(keeping as a draft while I do some extra testing and refactoring)

This PR adds a new step to the manual wizards of nodes and apps. DBs will be handled in a future issue.

Instead of just running `teleport start`, users will now run `teleport configure` to generate a config file and then run `teleport start --config file.yml`. You can see the reason to do this in [the issue](https://github.com/gravitational/teleport/issues/11740).

Note that this PR uses flags that are being added on https://github.com/gravitational/teleport/pull/11766, so we should not merge this PR before that one

Closes gravitational/teleport#11740

### CODE REVIEW

File: packages/teleport/src/Apps/AddApp/Manually/Manually.tsx [hunk 1]
```diff
@@ -77,6 +77,14 @@ export default function Manually({
   );
 }
 
+const configDir = '$HOME/.config';
```
[the remark this answers is not in the document]
Comment (hatched) [reply packages/teleport/src/Apps/AddApp/Manually/Manually.tsx:80]: Yeah something more teleport related.... `$HOME/.teleport`

File: packages/teleport/src/Nodes/AddNode/Manually/Manually.tsx [hunk 2]
```diff
@@ -80,6 +81,14 @@ export default function Manually({
   );
 }
 
+const configDir = '$HOME/.config';
+const configFile = `$HOME/.config/teleport.yaml`;
```
Comment (hatched) [inline packages/teleport/src/Nodes/AddNode/Manually/Manually.tsx:85]: Can you abstract these out into a shared location somewhere? I think we should probably set an app level default config output location.

I'm also wondering if we shouldn't use generated config file names as if they run this configure multiple times for apps, nodes, etc it'll override this config.

File: packages/teleport/src/Apps/AddApp/Manually/Manually.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcbattirola) [reply packages/teleport/src/Apps/AddApp/Manually/Manually.tsx:80]: I'll bring it up in the next UI meeting! TY

File: packages/teleport/src/Nodes/AddNode/Manually/Manually.tsx (hunk 2 above)
Comment (mcbattirola) [reply packages/teleport/src/Nodes/AddNode/Manually/Manually.tsx]: Hey, great ideas, I've added them in this commit https://github.com/gravitational/webapps/pull/751/commits/461a3f64a5eb1e0739583629bd44e227c5161243

**Review by kimlisa — COMMENTED** [review]
ran into a bug with the `server add`, try creating a token with the `iam` then go to another tab probably regression from the last PR

**Comment by mcbattirola** [comment]
> ran into a bug with the `server add`, try creating a token with the `iam` then go to another tab probably regression from the last PR committed

Thanks for noticing, opened a PR to fix it to master and backports to v8 and v9

**Review by kimlisa — APPROVED** [review]

**Review by hatched — APPROVED** [review]

**Pull request merged**